recipes/xqilla/all/patches/0002-support-cpp17.patch

```diff
@@ -0,0 +1,157 @@
+--- a/include/xqilla/ast/XQDocumentOrder.hpp
```

Comment:
It is better to raise a ``ConanInvalidConfiguration`` if this library cannot be built with C++17, than becoming maintainers of the library via patches.

Reply:
The latest version is from 2018, and raising ConanInvalidConfiguration is a pain for consumers.

Reply:
Yes, trying to consume a library that is not longer maintained is a pain for consumers. If the library wont' work with C++17, it is not the responsibility of the package manager to keep maintaining the library in the package manager side. This means becoming the maintainer of the library by proxy, and this is something that ConanCenter is not responsible for, and a liability that we will not assume.

Please remove that patch and raise a ``ConanInvalidConfiguration`` for C++17 and higher.

Reply:
I'll close this PR since I disagree with this new conan-center policy. Feel free to reopen a PR.